### ISSUE TITLE
Fatal error: SessionHandler::create_sid(): Cannot call default session handler in Unknown on line 0

### ISSUE THREAD

**Issue by GasparSukk** [opencart/opencart#5088]
Sorry, i have to make this ticket again. It is not resolved unfortunately.
Old one (closed prematurely): https://github.com/opencart/opencart/issues/5074
I made this fix in my code.
Now both areas - admin and frontend (same PHP version in use) - display this error.

OC 2.3.0.2
PHP 5.6.28
*******
```
Session Support enabled
Registered save handlers files user redis
Registered serializer handlers php_serialize php php_binary
session.auto_start	Off
session.cache_expire	180
session.cache_limiter	nocache
session.cookie_domain	no value
session.cookie_httponly	Off
session.cookie_lifetime	0
session.cookie_path	/
session.cookie_secure	Off
session.entropy_file	no value
session.entropy_length	0
session.gc_divisor	100
session.gc_maxlifetime	1440
session.gc_probability	1
session.hash_bits_per_character	4
session.hash_function	0
session.name	PHPSESSID
session.referer_check	no value
session.save_handler	files
session.save_path	/***/***/tmp (* the directory is writable by apache)
session.serialize_handler	php
session.upload_progress.cleanup	On
session.upload_progress.enabled	On
session.upload_progress.freq	1%
session.upload_progress.min_freq	1
session.upload_progress.name	PHP_SESSION_UPLOAD_PROGRESS
session.upload_progress.prefix	upload_progress_
session.use_cookies	On
session.use_only_cookies	On
session.use_strict_mode	Off
session.use_trans_sid	0
```

**Comment by GasparSukk** [comment]
The old version 2.2.0.0 works.

**Comment by arnisjuraga** [comment]
By the way, wild guess - do you have Compression enabled? 
`System > Settings > edit > Server tab`

`Output Compression Level` - is it something more than 0?

**Comment by GasparSukk** [comment]
It has set to 0.

**Comment by b3rend** [comment]
Any ideas? I have the exact same issue and a fix a nowhere to be found.

**Comment by ayoeng** [comment]
I have the same problem with php7. I can not install the oc

**Comment by danielkerr** [comment]
fixed in 3.0

just replace and createsid() lines with 

			if (function_exists('random_bytes')) {
				$session_id = substr(bin2hex(random_bytes(26)), 0, 26);
			} else {
				$session_id = substr(bin2hex(openssl_random_pseudo_bytes(26)), 0, 26);
			}
